superset/forms.py

```diff
@@ -50,7 +50,9 @@ def filter_not_empty_values(value):
 class CsvToDatabaseForm(DynamicForm):
     # pylint: disable=E0211
     def all_db_items():
-        return db.session.query(models.Database)
```

Comment:
It might be a good idea to rename this method as it's no longer _all__.

Reply:
Agree. Fixed